src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -27,6 +27,7 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 		vaccinationValueSetsProvider: VaccinationValueSetsProviding,
 		healthCertificateValidationOnboardedCountriesProvider: HealthCertificateValidationOnboardedCountriesProviding,
 		qrScannerCoordinator: QRScannerCoordinator,
+		badgeWrapper: HomeBadgeWrapper,
```

Comment:
```suggestion
		homeBadgeWrapper: HomeBadgeWrapper,
```

Reply:
Actually we are using the **badgeWrapper** in other part of the app that's why keep the same naming.

Reply:
Okay, your choice. It's just a tasty thing ;)